esphome/components/pm1006/sensor.py

```diff
@@ -16,7 +16,7 @@
 DEPENDENCIES = ["uart"]
 
 pm1006_ns = cg.esphome_ns.namespace("pm1006")
-PM1006Component = pm1006_ns.class_("PM1006Component", uart.UARTDevice, cg.Component)
+PM1006Component = pm1006_ns.class_("PM1006Component", uart.UARTDevice, cg.PollingComponent)
```

Comment:
I think this is what black wants
```suggestion
PM1006Component = pm1006_ns.class_(
    "PM1006Component", uart.UARTDevice, cg.PollingComponent
)
```